### ISSUE TITLE
[0.9.0 Staging - 1526] Rebalancing Asphalt Roads

### ISSUE THREAD

**Issue by Sarah-Graesyn** [StrangeLoopGames/EcoSuggestions#1082]
When we first started the Playtest for 9.0 Asphalt roads had a requirement for quicklime I believe it was in order to craft them that got removed at one point, making the jump from Stone Roads to Asphalt roads _extremely_ easy and basically nullifying the use of Stone Roads as a mid-tier road at all. 

I personally think that separating them out again by re-adding that quicklime (or perhaps even oil, _maybe both_) requirement to craft them would make the progression of technologies and such in Eco much much better and a lot more common sense.

**Comment by trickenso** [comment]
Maybe a some tagged resources? Generalise it to be an aggregate and a binder, rather than the single recipe there is currently?

This would allow modern roads to be built out of concrete or oil byproducts or plant polymers or similar as a binding agent - more flexible and interesting, while still retaining something more involved than the current stone, stone and more stone. Lots of room for future additions of bioplastics, epoxy or other binding agents if you wanted, as well.
This might also give some reprocessing uses for old brick blocks, for instance, if they could be crushed into brick aggregate, for instance - and slag is often used as a road aggregate.

I like the lime suggestion - given its real-life requirements, mightn't the concrete be a good candidate for requiring lime in some form?
> Portland cement is by far the most common type of cement in general use around the world. This cement is made by heating limestone (calcium carbonate) with other materials (such as clay) to 1,450 °C (2,640 °F) in a kiln
https://en.wikipedia.org/wiki/Cement#Portland_cement

I'm not necessarily advocating a total rebalance of concrete (the ability for cement kilns to consume stones in huge quantities is a lot of their appeal) but at the moment concrete is made without anything that might constitute a chemical binder or cause such a reaction. Even requiring crushed limestone would make some sense...

Using quicklime in concrete might also link cement making with advanced smelting, which could be an interesting opportunity to increasing collaboration.

**Comment by Sarah-Graesyn** [comment]
@trickenso yes, this is kind of what I am getting at here. And when we started the playtest there was a quicklime requirement for asphalt roads but it got removed during one of the rebalance builds and the removal just makes it too easy to get to Asphalt.

I really like the idea of using another tag, though perhaps not plant polymers though then again I have seen some interesting things get used to create the roads in our real world so, you never know. I'd love to see some oil byproducts and quicklime used in their creation and see an expansion of different skills being used in order to achieve this final step of roads.

**Comment by trickenso** [comment]
> I really like the idea of using another tag, though perhaps not plant polymers though then again I have seen some interesting things get used to create the roads in our real world so, you never know.

Maybe the tag could just be 'binder' and you could use oil products or concrete or plant polymers or whatever you like that fits the category.

The idea of one community being oil-guzzling and making conventional asphalt roads versus another being almost entirely plant-based and using manufactured plant polymers would be an interesting contrast.
